src/mongo/db/query/explain.cpp

```diff
@@ -280,6 +281,30 @@ std::vector<std::unique_ptr<PlanStageStats>> getRejectedPlansTrialStats(PlanExec
     return res;
 }
 
+/**
+ * Gather the PlanStages when a backup plan is used
```

Comment:
This comment is confusing. What do we do when a backup plan is not used? 

I'd actually recommend removing this method, and instead adding a 'bool backupPlanUsed' parameter to 'getRejectedPlansTrialStats'.

Reply:
If you do that, you should then extend the comment on that method to include something like "If a backup plan was used, include the stats of the new winning plan instead of the original winning plan. The original winning plan is expected to already be known by the caller, since it needs to be snapshotted and stored before execution completes."